src/components/ArticlePreview.jsx

```diff
@@ -3,31 +3,42 @@ import { Link } from 'react-router';
 
 export default class ArticlePreview extends React.Component {
 
-  loadPost(e) {
-    e.preventDefault();
-    // LOAD_ARTICLE action
-  }
-
   render () {
     var article = this.props.article;
+
+    // TODO: ensure ArticlePreview can handle multiple authors
+    // TODO: edit link routing and remove /0/
     return (
       <div>
-        <img src={article.image} alt="featured" />
-        // TODO: ensure ArticlePreview can handle multiple authors
-        <Link to={`/$(article.author)`}>{article.author}</Link>
-        <Link to={`/$(article.issue)/$(article.slug)`}>{article.title}</Link>
+        <div>
```

Comment:
Same here. unless you are using these extra divs for layouting, see if you can delete them.